libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -220,7 +286,7 @@ static void vTaskRxSelect( void * param )
             /*ctx->rx_callback = NULL; */
```

Comment:
Please delete commented-out code on lines 286, 288, and 296 while we're in here.